static/src/stylesheets/module/facia/_icons.scss

```diff
@@ -138,66 +118,59 @@
 .tone-comment--item,
 .tone-dead--item,
 .tone-news--item {
-    .fc-sublink--video .fc-sublink__link:before {
-        @include icon(video-icon-black, false);
+    .fc-sublink--video .fc-sublink__link .inline-icon {
+        fill: #000000;
     }
 
-    .fc-sublink--gallery .fc-sublink__link:before {
-        @include icon(camera-black-large, false);
+    .fc-sublink--gallery .fc-sublink__link .inline-icon {
+        fill: #000000;
     }
 
-    .fc-sublink--audio .fc-sublink__link:before,
-    .fc-sublink--audio.tone-podcast--sublink .fc-sublink__link:before {
-        @include icon(volume-high-black, false);
+    .fc-sublink--audio .fc-sublink__link path,
+    .fc-sublink--audio.tone-podcast--sublink .fc-sublink__link .inline-icon {
+        fill: #000000;
```

Comment:
can the `fill` inherit from `.fc-sublink--audio.tone-podcast--sublink`, save a few selectors?